demos/sslecho/echecho.c

```diff
@@ -0,0 +1,384 @@
+/*
+ *  Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Year is wrong :(

Reply:
I put in "2022-2024" there (and in various others)

Reply:
New files should have 2024 I think.

Reply:
sure, did that (might've missed some but we'll catch 'em as we go, and hopefully it'll not be 2025 before we're done:-)